src/it/scala/com/exasol/spark/BaseTableQueryIT.scala

```diff
@@ -0,0 +1,35 @@
+package com.exasol.spark
+
+class BaseTableQueryIT extends AbstractTableQueryIT {
+
+  val schema = "TEST_SCHEMA"
+  override val tableName: String = s"$schema.TEST_TABLE"
+  override def createTable: Unit =
+    connectionManager.withExecute(
+      // scalastyle:off nonascii
+      Seq(
```

Comment:
Yes, wanted to change but it cause some problems because of the default values (`identity` and `current_timestamp`).

TDDB expects them from user when inserting. I am going to address it in another ticket #90.

Reply:
ok